spec/support/view_component/context.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Support
+  module ViewComponent
+    class Context
+      StubbedComponent = Struct.new(:instance, :args, :output)
+
+      def register_component(instance, args)
+        components << StubbedComponent.new(
+          instance,
+          args,
+          Faker::Lorem.paragraphs.join(" "),
+        )
+      end
+
+      def output_for(instance)
+        components.find { |comp| comp.instance == instance }.output
+      end
+
+      def components
+        @components ||= []
+      end
```

Comment:
When is this wiped? Every test? Every suite? Every test run?

Reply:
Each test has it's own context. :)